{"name": "输入错误用户名正确密码登录失败", "status": "passed", "attachments": [{"name": "stdout", "source": "a4278fcd-09a6-4773-9548-81031b3fe928-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '6', 'casename': '输入错误用户名正确密码登录失败', 'mode': '登录', 'data': 'userAccount=979172251@qq.commm\\npassword=a123456', 'assert_way': 'class name=login-form-forgot', 'result': '\\n              Forgot Password?\\n            '}"}], "start": 1650344593452, "stop": 1650344605782, "uuid": "a9842410-8791-4299-b87e-e640a4740f8b", "historyId": "b48564a7bfa65ccf6e5f01371b381708", "testCaseId": "f42af1ba93a36a6b78bc4dd2bc5b083c", "fullName": "test_main.Test_Main#test_login", "labels": [{"name": "feature", "value": "登录"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "12964-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}